### ISSUE TITLE
GetRecords POST does not accept an ElementSetName

### ISSUE THREAD

**Issue by andrew-wils** [geopython/pycsw#543]
# Description

Posting a GetRecords request with only an ElementSetName element triggers the following exception:

> <ows20:ExceptionText>Only ONE of ElementSetName or ElementName parameter(s) is permitted</ows20:ExceptionText>

If there is a problem with the request, it does not seem to be described by the above message.

# Environment

pycsw 2.1-dev on demo.pycsw.org

# Steps to Reproduce

HTTP POST the following:
```
<?xml version='1.0' encoding='UTF-8'?>
<csw30:GetRecords version="3.0.0" service="CSW" xmlns:csw30="http://www.opengis.net/cat/csw/3.0">
  <csw30:Query typeNames="csw30:Record">
        <csw30:ElementSetName>full</csw30:ElementSetName>
  </csw30:Query>
</csw30:GetRecords>
```

to 

http://demo.pycsw.org/cite/csw?request=GetRecords

**Comment by tomkralidis** [comment]
@andrew-wils thanks for the report. I've fixed this in master and 2.0 branch.

**Comment by kalxas** [comment]
I think this one can be closed now
